### PR TITLE
Remove integration tests

### PR DESCRIPTION
**Pull request by Gudahtt**
The remaining integration tests are all covered by e2e tests, so they're no longer needed.

All associated scripts, fixtures, and dependencies have also been removed.

### CODE REVIEW

**Comment by Gudahtt** [comment]
This depends upon #8958

**Review by brad-decker — DISMISSED** [review]
Heck yeah, @Gudahtt 👍

File: development/sourcemap-validator.js [hunk 1]
```diff
@@ -1,106 +0,0 @@
-const fs = require('fs')
```
Comment (Gudahtt) [inline development/sourcemap-validator.js:1]: Oops, this file was not supposed to be deleted. I'll restore it.

Comment (Gudahtt) [reply development/sourcemap-validator.js:1]: This has been restored.

**Review by rekmarks — APPROVED** [review]
image

You deserve this 🙏

**Review by whymarrh — APPROVED** [review]
I like this

**Pull request merged**